src/components/Login/Login.js

```diff
@@ -140,6 +140,7 @@ class Login extends Component {
   }
 
   signInFB() {
+    this.setState({ loaded: false });
```

Comment:
Oh, Thank you for letting me know that. I'm just wondering... do they have some difference in any perspective?